api/v1alpha1/storageconsumer_types.go

```diff
@@ -42,7 +43,23 @@ type StorageConsumerSpec struct {
 	Enable bool `json:"enable,omitempty"`
 	// StorageQuotaInGiB describes quota for the consumer
 	// +optional
-	StorageQuotaInGiB int `json:"storageQuotaInGiB,omitempty"`
+	StorageQuotaInGiB          int                            `json:"storageQuotaInGiB,omitempty"`
+	CephResourcesConfigMapRef  corev1.LocalObjectReference    `json:"cephResourcesConfigMapRef,omitempty"`
```

Comment:
The name here does not fit the intent of the configmap.
The content of the config map is not directly related to ceph resources but more to reconcile ownership.
Also, this field is optional be definition and we should add an +optional comment above it

Reply:
changed to `OwningResourcesConfigMapRef`